redis/asyncio/cluster.py

```diff
@@ -516,35 +517,44 @@ def set_response_callback(self, command: str, callback: ResponseCallbackT) -> No
 
     async def _determine_nodes(
         self, command: str, *args: Any, node_flag: Optional[str] = None
-    ) -> List["ClusterNode"]:
+    ) -> Tuple[List["ClusterNode"], bool]:
```

Comment:
Since `is_default_node` value will only be checked in the rare case of cluster failover or scaling/re-sharding, might it be better to compare the target_node with self.nodes_manager.default_node instead of changing the return signature? Even though this is an internal function, users/packages might still be overwriting it to add/modify functionality.

Reply:
Changed